packages/naming.presets/origin.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+    delims: {
+        elem: '__',
+        mod: { name: '_', val: '_' }
+    },
+    fs: {
+        // delims: { elem: '__', mod: '_' }, // redundand because of defaults
+        pattern: '{layer}.blocks/{entity}.{tech}',
```

Comment:
will it work for `design/common.blocks/button/_theme/button_theme_islands.post.css` ?

Reply:
It will work for `common.blocks/button/_theme/button_theme_islands.post.css` inside `design` level